runtime/bcutil/cfreader.c

```diff
@@ -1174,7 +1174,7 @@ readPool(J9CfrClassFile* classfile, U_8* data, U_8* dataEnd, U_8* segment, U_8*
 				goto _errorFound;
 			}
 
-			if (!J9_ARE_ALL_BITS_SET(classfile->accessFlags, CFR_ACC_MODULE)) {
+			if (classfile->accessFlags != CFR_ACC_MODULE) {
```

Comment:
generally, we do bit operations/checks on`accessFlags`. better to add this comment over here as well:
```
	/* Java 9 Spec: If the ACC_MODULE flag is set in the access_flags item, then no
	 * other flag in the access_flags item may be set. If other flags are set, then this
	 * file is not a module.
```